src/Widgets/Banner.vala

```diff
@@ -61,7 +61,13 @@ namespace AppCenter.Widgets {
 
                 string description;
                 if (has_package) {
-                    description = package.get_description ();
+                    string[] lines = package.get_description ().split ("\n");
+                    description = lines[0].strip ();
```

Comment:
Can the description be empty in rare cases? If so, that will need to be guarded against to avoid a crash?

Reply:
If it's featured on the homepage it should definitely have a description since it would be reviewed